pkg/virt-config/config_test.go

```diff
@@ -366,17 +367,17 @@ var _ = Describe("ConfigMap", func() {
 		resultJson, err = json.Marshal(kubevirtConfig)
 		Expect(err).ToNot(HaveOccurred())
 		Expect(string(kubevirtConfigJson)).To(BeEquivalentTo(string(resultJson)))
-
 	},
 		table.Entry("when values set, should equal to result",
 			`{"machineType":"test","cpuModel":"test"}`,
 			v1.KubeVirtConfiguration{CPURequest: &defaultCPURequest, MachineType: "test", CPUModel: "test"}),
 		table.Entry("when networkConfigurations set in kubevirt.yaml, should equal to result",
-			`{"network":{"defaultNetworkInterface":"test","permitSlirpInterface":"true","permitBridgeInterfaceOnPodNetwork":"true"}}`,
-			v1.KubeVirtConfiguration{CPURequest: &defaultCPURequest, NetworkConfiguration: &v1.NetworkConfiguration{NetworkInterface: "test", PermitSlirpInterface: true, PermitBridgeInterfaceOnPodNetwork: true}}),
+			`{"network":{"defaultNetworkInterface":"test","permitSlirpInterface":"true","permitBridgeInterfaceOnPodNetwork":"false"}}`,
```

Comment:
I changed this to make sure that when the value false is set that the object also sets the value to false. This is why in the object this was changed to a bool ptr instead of a bool, otherwise the value would always be true since we didn't know the difference between the value being not set and we should use the default and the value being purposely set to false.